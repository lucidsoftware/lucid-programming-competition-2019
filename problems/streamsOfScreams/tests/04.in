5
boo!
boo!
boo!
boo!
boo!