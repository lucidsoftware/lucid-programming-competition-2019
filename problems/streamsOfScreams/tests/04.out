Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
Ahhh!!!
15